qemu/tests/cfg/balloon_boot_in_pause.cfg

```diff
@@ -0,0 +1,34 @@
+- balloon_boot_in_pause: install setup image_copy unattended_install.cdrom
+    virt_test_type = qemu
+    no Win2000, Fedora.8, Fedora.9, Fedora.10, RHEL.3, RHEL.4, Unix, livecd
+    type = balloon_boot_in_pause
+    balloon = balloon0
+    balloon_dev_devid = balloon0
+    balloon_dev_add_bus = yes
+    ratio_1 = 0.8
+    ratio_2 = 0.5
```

Comment:
As we always start vm with more than 2G memory, I think it is acceptable for our tests.